cmd/pd-server/main.go

```diff
@@ -41,6 +41,11 @@ import (
 	"github.com/tikv/pd/server/config"
 	"github.com/tikv/pd/server/join"
 	"go.uber.org/zap"
+
+	// register microservice HTTP API
+	_ "github.com/tikv/pd/pkg/mcs/resourcemanager/server/apis/v1"
```

Comment:
why need this? Does `install` pkg duplicate with it?

Reply:
If we don't import, the independent service cannot use the HTTP interface in the current implementation. There is an import cycle problem.

Reply:
I mean why not 
```
_ "github.com/tikv/pd/pkg/mcs/resourcemanager/server/install"
_ "github.com/tikv/pd/pkg/mcs/scheduling/server/install"
_ "github.com/tikv/pd/pkg/mcs/tso/server/apis/install"
```

make http and grpc register togeter. it exists import cycel? I think it's same with xxx/api.

Reply:
fixed